test/test_estimate_fee.py

```diff
@@ -30,6 +30,33 @@ def send_estimate_fee_with_requests(req_dict: dict):
         json=req_dict
     )
 
+def common_estimate_response(response):
+    """expected response from estimate_fee request"""
+    assert response.status_code == 200
+    response_parsed = response.json()
+
+    assert response_parsed.get("gas_price") == GAS_PRICE
+    assert isinstance(response_parsed.get("gas_usage"), int)
+    assert response_parsed.get("overall_fee") == response_parsed.get("gas_price") * response_parsed.get("gas_usage")
+    assert response_parsed.get("unit") == "wei"
+    assert response_parsed.get("warning") is None
```

Comment:
We don't need the warning assertion anymore, right?

Reply:
9fcf2d526c62fe1dbc5112dd250fe54bc3727fe8